### DISCUSSION TITLE
Access Developer tools after wails build

### DISCUSSION THREAD

**Discussion by brody192** [wailsapp/wails#1526] [Q&A] [answered]
I have an app (vanilla JS + vite) it works fine during "wails dev" but after it's built Javascript seems not to be running (probably some trivial error holding it up), strangely enough, the web app would work if I hosted the dist folder from a web server. I'm positive if I could manage to see a console log of the built app I could debug it, any help with getting a console log from the built app would be appreciated! Thanks!

**Comment by leaanthony** [comment, marked as answer]
`wails build -debug`. I've just updated the docs to state this which should be live in about 10 mins. Good luck! 👍

**Comment by brody192** [comment]
And with that it's fixed, the problem was `//go:embed frontend/src` instead of dist, this project used to not use any build step. Thanks again!
